charts/datadog/templates/_helpers.tpl

```diff
@@ -150,3 +150,28 @@ true
 false
 {{- end -}}
 {{- end -}}
+
+{{/*
+Return probe definition based on user settings and default HTTP port
+*/}}
+{{- define "probe.http" -}}
+{{- if or .settings.httpGet .settings.tcpSocket .settings.exec -}}
```

Comment:
Where is `.settings` coming from?

Reply:
Here's on example: https://github.com/DataDog/helm-charts/pull/69/files#diff-a80e1dad44b204bcd75eed397bfa4a89f944c4804baebd6f988392094f058dc2R205, argument to a helper is a dict.

Reply:
Okay understood. I think it's worth adding it in the comment

Reply:
Added a comment.